event.go

```diff
@@ -45,7 +45,9 @@ type EventBuilder struct {
 	// The events needed to authenticate this event.
 	AuthEvents []EventReference `json:"auth_events"`
 	// The event ID of the event being redacted if this event is a "m.room.redaction".
-	Redacts  string `json:"redacts,omitempty"`
+	Redacts string `json:"redacts,omitempty"`
+	// The depth of the event.
```

Comment:
It might be nice to state what this depth relates to? Depth from the `m.room.create` event?